source/support.html.erb

```diff
@@ -22,6 +22,7 @@ title: Support - GOV.UK Pay
       <h1 class="heading-large">Support</h1>
       <p>GOV.UK Pay is built and supported 24/7 by a full-time team at the Government Digital Service. The service manager is Till Wirth.</p>
       <p>Follow our <a href="/getstarted/" data-click-events data-click-category="Content" data-click-action="Internal link clicked">guide to getting started</a> to create an account.</p>
+      <p>GOV.UK Pay's support is for government services, and does not provide support for individual members of the public paying for services. If you are a member of the public experiencing technical difficulties paying for a specific service, please contact that service's support directly.</p> 
```

Comment:
Could you match the rest of the file by using curly apostrophes like ’ rather than straight ones like ' please? (`Shift`+`Alt`+`]` on a British English Mac keyboard)